src/svg/ellipse-shape-gray.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="650" height="454" viewBox="0 0 650 454" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Can you remove this  `.svg` it's not going to be needed I will had the ellipse component to this page afterwards 👍